pygments/lexers/usd.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python
```

Comment:
> use the standard file header

Sure, you mean how the other modules just define encoding and don't provide a shebang line, right? Done

> import conventions (e.g. import tokens as from token import Punctuation)

I can't tell if you're suggesting to

- not use `tokens.Tokens.Foo` and instead use the re-defined names as `tokens.Foo`
- change the imports to absolute 
- use imports of class / function names, instead of module names.

or some combination of the 3. The first item is my bad and is changed. But the other two I think you should reconsider. (I'm not trying to start an argument or anything, just saying you should consider it).

Reply:
For the file header, I meant the standard docstring with module name etc. But that's a detail, I'll add that post merging.

About the imports I'm sorry I was a bit vague. But your latest version is what I wanted, so I'm happy.

Reply:
:+1: